### PR TITLE
New quotes

### PR DESCRIPTION
**Pull request by williamsaaron**

### CODE REVIEW

File: src/quotes.jade [hunk 1]
```diff
@@ -90,6 +94,10 @@ block content
         p <strong>Alois Reitbauer</strong><br>Chief Technology Strategist, <a href="http://www.dynatrace.com/" target="_blank">Dynatrace</a>
         img(src='/assets/images/logos/partners/dynatrace.png').quote-image
       .quote-block
+        p "We are excited for the launch of DC/OS ushering in a new dawn for Big Data. DC/OS provides interoperable service discovery and load balancing via Minuteman which is vital to the Software Defined Data Center's success. Elodina's Platform is plug and play with DC/OS allowing companies building an internal PaaS to orchestrate and operate Apache Kafka, Apache Storm, Apache Hadoop, Apache Accumulo, Apache Cassandra, Zipkin, StatsD, SysLog, ElasticSearch, DataStax Enterprise Edition and many more Frameworks as a Service. Elodina & DC/OS clusters provide multi-tenancy and micro-segmentation with split second auto scaling response times, auction bidding for resources and infrastructure portability for streams and storage of Giga, Tera, Peta and Exa bytes of production data."
```
Comment (karlkfi) [inline src/quotes.jade]: might need `&amp;` instead of just `&`. Not sure if jade escapes it.

Comment (williamsaaron) [inline src/quotes.jade]: Yeah, I was nervous the first time too, but there are already several `&` in the doc by now, they render fine.

**Comment by karlkfi** [comment]
this ready to go?

**Comment by williamsaaron** [comment]
Yes please, thanks @karlkfi.

**Pull request merged**